ipv8/src/main/java/nl/tudelft/ipv8/Community.kt

```diff
@@ -43,11 +41,18 @@ abstract class Community : Overlay {
     private lateinit var job: Job
     protected lateinit var scope: CoroutineScope
 
+    var evaProtocolEnabled = false
```

Comment:
I think it would make sense for this to be moved to the Peer class, where `supportsTFTP` also resides.

Reply:
The ```supportsTFTP``` is Peer dependent while the ```evaProtocolEnabled``` is community dependent and not Peer dependent. If the community has it enabled all peers will use it.